src/notification.c

```diff
@@ -403,6 +407,22 @@ on_window_map (GtkWidget  *widget, GdkEvent   *event, WindowData *windata) {
     return FALSE;
 }
 
+void on_size_allocate(GtkWidget *widget, GtkAllocation *allocation, WindowData *windata) {
+    Settings settings = windata->settings;
+    if(settings.pos_x > 0 || settings.pos_y > 0) {
```

Comment:
Is the intended behaviour to center the window if either of the two positions is 0, or if both are set to zero? Also, what happens if I want to position my window at 0, 0? How does this all play out with the `settings.center` variable?

Reply:
Fixed this, now if `x=0,y=0, center=false`, the notification will appear in the left side of the screen.